WordPress/gutenberg
# [RNMobile] Page Template Picker UI Improvements

`Gutenberg-mobile` PR -> https://github.com/wordpress-mobile/gutenberg-mobile/pull/1720

## Description
This PR updates the current styles for the Page Template picker. It also includes some code changes to support more items in the list and accessibility.

## To test
- Open WP iOS / WP Android
- Go to **Site Pages**
- Add a new page
  - For iOS Tap on the Add button in the header (top right)
- **Expect** the page template picker to be shown with the styles from the screenshots
- Tap on either predefined layout
- **Expect** the content of the page to change and the picker to **disappear**

| iOS | Android |
|     :---:      |     :---:      |
| <img src="https://user-images.githubusercontent.com/4885740/71714968-13841880-2e10-11ea-9339-2c9de1b4c513.gif" width=250 /> | <img src="https://user-images.githubusercontent.com/4885740/71714988-27c81580-2e10-11ea-8ddc-d5d0db6855ad.gif" width=270 />  |

## Screenshots

### iOS 

| Default | Dark mode |
|     :---:      |     :---:      |
| <img src="https://user-images.githubusercontent.com/4885740/71715020-40383000-2e10-11ea-98a1-354d7c6e752f.png" width=250 /> | <img src="https://user-images.githubusercontent.com/4885740/71715023-4201f380-2e10-11ea-8199-bb864888657f.png" width=250 />  |

| iPad Default | iPad Dark mode |
|     :---:      |     :---:      |
| <img src="https://user-images.githubusercontent.com/4885740/71715039-580fb400-2e10-11ea-8b82-ef4d59540369.png" width=250 /> | <img src="https://user-images.githubusercontent.com/4885740/71715041-59d97780-2e10-11ea-84d0-270a399b338b.png" width=250 />  |

### Android

<img src="https://user-images.githubusercontent.com/4885740/71715070-6cec4780-2e10-11ea-9a10-402520419053.png" width=250 /> 

## Types of changes
- UI Styling
- Accessibility

## Checklist:
